server/src/internalClusterTest/java/org/opensearch/indices/IndicesRequestCacheIT.java

```diff
@@ -677,14 +689,324 @@ public void testCacheWithInvalidation() throws Exception {
         assertCacheState(client, "index", 1, 2);
     }
 
+    // when staleness threshold is low, it should clean the cache
+    public void testStaleKeysCleanup_LowThresholdCleansUpCache() throws Exception {
+        Instant start = Instant.now();
+        long thresholdInMillis = 1_500;
+        String node = internalCluster().startNode(
```

Comment:
start data only node?

Reply:
Yes, this starts a single node cluster only to be used for this specific test case.

Reply:
that node will be a dn and a cluster manager